src/main/java/org/mockito/internal/creation/settings/CreationSettings.java

```diff
@@ -170,6 +171,11 @@ public boolean isStubOnly() {
 
     @Override
     public boolean isLenient() {
```

Comment:
Do we still use this method somewhere else? If yes, we should refactor these to call `getStrictness` and deprecate this method (since we ship it in the `MockSettings` interface, we can't remove it just yet).

Reply:
Yes, this isLenient() is being used in one place in main code. I have refactored that to use getStrictness().

But this isLenient() is also being used in few tests, I think we can remove those reference while removing this deprecated method.